data/random-teams.js

```diff
@@ -1881,32 +1881,52 @@ class RandomTeams extends Dex.ModdedDex {
 		}
 
 		let items = [];
-		if (Array.isArray(setData.set.item) === true) {
+		if (Array.isArray(setData.set.item)) {
 			let randomItem = setData.set.item;
 			items.push(setData.itemVariants ? randomItem[setData.itemVariants] : this.sample(randomItem));
 		} else {
 			items.push(setData.set.item);
 		}
 
 		let abilities = [];
-		if (Array.isArray(setData.set.ability) === true) {
+		if (Array.isArray(setData.set.ability)) {
 			let randomAbility = setData.set.ability;
 			abilities.push(setData.abilityVariants ? randomAbility[setData.abilityVariants] : this.sample(randomAbility));
 		} else {
 			abilities.push(setData.set.ability);
 		}
 
+		let evs = {};
+		if (setData.set.evs) {
+			for (let ev in setData.set.evs) {
+				if (!setData.set.evs[ev]) evs[ev] = 0;
+				evs[ev] = setData.set.evs[ev];
```

Comment:
Missing an `else`?

Reply:
no

Reply:
Might as well do the same thing with this, i.e. `Object.assign({hp: 0, atk: 0, def: 0, spa: 0, spd: 0, spe: 0}, setData.set.evs);` If there are EVs missing in future set updates it'll probably be easier to notice if they aren't 84 across the board instead.